outline/manager.js

```diff
@@ -381,7 +381,7 @@ module.exports = function (doc) {
   this.showDashboard = showDashboard
 
   function expandedHeaderTR (subject, requiredPane, options) {
-    function renderPaneIconTray (td) {
+    function renderPaneIconTray (td, hideList) {
```

Comment:
To avoid making our API sprawl (`renderPaneIconTray(td, false, true, undefined, null, true)`) even worse, maybe make the second parameter an `options` object?

Reply:
I agree with the need to avoid API sprawl, but perhaps add an `options` object when we more parameters? Feels a bit weird to do for one property.

Reply:
But that means breaking the API, which is especially painful in a file that is not written in TypeScript :) (i.e. you'd not only have to update the parameters here, but also everwhere this function is called - though I guess that might be only in this file?)

Reply:
Ah, right, that's a good argument. Ok, I'll make it into an option-parameter.